mailpile/crypto/aes_utils.py

```diff
@@ -99,7 +99,10 @@ def aes_ctr_decryptor(key):
 try:
     aes_ctr_encryptor, aes_ctr_decryptor = make_cryptography_utils()
 except ImportError:
-    aes_ctr_encryptor, aes_ctr_decryptor = make_pycrypto_utils()
+    try:
+        aes_ctr_encryptor, aes_ctr_decryptor = make_pycrypto_utils()
+    except ImportError:
+        raise ImportError("Please pip install cryptography (or pycrypto)")
```

Comment:
could be more verbose, like, "Couldn't find python 'cryptography' module, nor fallback 'pycrypto' module" ?  but yeah, this is already going to be better. :)